proto/messages.proto

```diff
@@ -206,7 +207,8 @@ message Asset {
 // Actual message is encrypted with AES and sent as additional data
 message External {
   required bytes otr_key = 1;
-  optional bytes sha256 = 2;      // sha256 of ciphertext
+  optional bytes sha256 = 2;      // sha256 of ciphertext, obsolete but required for backward compatibility
```

Comment:
This is correct. But in fact we want to deprecate AES_CBC, and there is no need to compute a separate SHA for GCM according to @raphaelrobert.